types/gas.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	storetypes "cosmossdk.io/store/types"
+)
+
+// Wrapper errors for store/v1 ErrorOutOfGas, ErrorNegativeGasConsumed and ErrorGasOverflow so that
+// modules don't have to import storev1 directly.
```

Comment:
I'd rather have module import store tan the SDK tbh (my t.wo cents)
Can we avoid adding stuff in the SDK?

Reply:
hmmmm if we use stf gas implementation then we can avoid import storetypes ErrorOutOfGas all together (so i dont have to wrapped it here). But then we need to add a wrapper for the legacy sdkCtx gas meter, the problem is that the wrapper is in runtime